phalcon/di/exception/serviceResolutionException.zep

```diff
@@ -0,0 +1,30 @@
+
+/*
+ +------------------------------------------------------------------------+
+ | Phalcon Framework                                                      |
+ +------------------------------------------------------------------------+
+ | Copyright (c) 2011-2017 Phalcon Team (https://phalconphp.com)          |
+ +------------------------------------------------------------------------+
+ | This source file is subject to the New BSD License that is bundled     |
+ | with this package in the file LICENSE.txt.                             |
+ |                                                                        |
+ | If you did not receive a copy of the license and are unable to         |
+ | obtain it through the world-wide-web, please send an email             |
+ | to license@phalconphp.com so we can send you a copy immediately.       |
+ +------------------------------------------------------------------------+
+ | Authors: Andres Gutierrez <andres@phalconphp.com>                      |
+ |          Eduar Carvajal <eduar@phalconphp.com>                         |
+ +------------------------------------------------------------------------+
+ */
+
+namespace Phalcon\Di\Exception;
+
+use Phalcon\Di\Exception\ServiceResolutionException;
+
+/**
+ * Phalcon\Di\Exception\ServiceResolutionException
+ *
+ */
+class ServiceResolutionException extends \Phalcon\Di\Exception
+{
+}
```

Comment:
Could you rename this file `phalcon/di/exception/serviceResolutionException.zep` => `phalcon/di/exception/serviceresolutionexception.zep` to follow common style